apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/PartitionOptionsTests.cs

```diff
@@ -31,40 +31,7 @@ public void CheckDefaultValues()
     }
 
     [Fact]
-    public void Default_WithDataBoostEnabled_ShouldCreateNewInstanceWithUpdatedValue()
-    {
-        var options = PartitionOptions.Default.WithPartitionSizeBytes(10).WithMaxPartitions(20).WithDataBoostEnabled(false);
-        var newOptions = options.WithDataBoostEnabled(true);
-
-        Assert.Equal(10, newOptions.PartitionSizeBytes);
-        Assert.Equal(20, newOptions.MaxPartitions);
-        Assert.Equal(true, newOptions.DataBoostEnabled);
-    }
-
-    [Fact]
-    public void Default_WithMaxPartitions_ShouldCreateNewInstanceWithUpdatedValue()
-    {
-        var options = PartitionOptions.Default.WithPartitionSizeBytes(10).WithMaxPartitions(20).WithDataBoostEnabled(false);
-        var newOptions = options.WithMaxPartitions(200);
-
-        Assert.Equal(10, newOptions.PartitionSizeBytes);
-        Assert.Equal(200, newOptions.MaxPartitions);
-        Assert.Equal(false, newOptions.DataBoostEnabled);
-    }
-
-    [Fact]
-    public void Default_WithPartitionSizeBytes_ShouldCreateNewInstanceWithUpdatedValue()
-    {
-        var options = PartitionOptions.Default.WithPartitionSizeBytes(10).WithMaxPartitions(20).WithDataBoostEnabled(false);
-        var newOptions = options.WithPartitionSizeBytes(100);
-
-        Assert.Equal(100, newOptions.PartitionSizeBytes);
-        Assert.Equal(20, newOptions.MaxPartitions);
-        Assert.Equal(false, newOptions.DataBoostEnabled);
-    }
-
-    [Fact]
-    public void CheckOptionsAreImmutable()
+    public void CheckWithMethods()
```

Comment:
Again, I'd just rename it to `WithMethods` - we're in a test file, so the "check" aspect can be taken for granted.